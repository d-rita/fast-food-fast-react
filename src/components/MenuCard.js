/* eslint-disable camelcase */
import React from 'react';

const MenuCard = (props) => {
  const token = localStorage.getItem('token');
  const addButton = token ? <button type="button" className="addButton">Add food</button> : null;
  const { food } = props;
  const { food_name, food_price } = food;
  return (
    <div className="card">
      <div className="card-body row">
        <div className="col-sm-3">
          <h5>{food_name}</h5>
        </div>
        <div className="col-sm-3">
          <h5>{food_price}</h5>
        </div>
        <div className="col-sm-6">
          {addButton}
        </div>
      </div>
    </div>
  );
};

export default MenuCard;
