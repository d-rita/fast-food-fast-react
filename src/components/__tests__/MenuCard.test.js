import React from 'react';
import { shallow } from 'enzyme';
import MenuCard from '../MenuCard';


describe('Menu Card', () => {
  let props;
  it('should render without crashing', () => {
    props = {
      food: {
        food_name: 'chicken',
        food_price: 12000
      }
    };
    const wrapper = shallow(<MenuCard {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
