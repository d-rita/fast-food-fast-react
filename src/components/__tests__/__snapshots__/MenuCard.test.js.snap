// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Card should render without crashing 1`] = `ShallowWrapper {}`;
